packages/core/src/AttesterSchema.ts

```diff
@@ -0,0 +1,278 @@
+import { CircuitConfig } from '@unirep/circuits'
+
+const { FIELD_COUNT, EPOCH_TREE_DEPTH, SUM_FIELD_COUNT, REPL_NONCE_BITS } =
+    CircuitConfig.default
+class DataField {
+    public field: string[]
+    public fieldOffset: number[]
+    public currentFieldSize: bigint
+
+    private maxFieldSize: number
+
+    constructor() {
+        this.field = []
+        this.fieldOffset = []
+        this.currentFieldSize = BigInt(0)
+
+        this.maxFieldSize = 254
+    }
+
+    public addOrPass(entry: string, size: bigint): boolean {
+        if (this.currentFieldSize + size > this.maxFieldSize) {
```

Comment:
if `this.maxFieldSize` is a `number` type, I think `this.currentFieldSize` and `size` don't have to be `bigint` type

Reply:
Fixed - using number now